easybuild/easyconfigs/d/DOLFINx-MPC/DOLFINx-MPC-0.9.0-foss-2023b.eb

```
@@ -0,0 +1,54 @@
+# Authors:: Tanmoy Chakraborty (University of Warwick/tanmoy.chakraborty@warwick.ac.uk)
+
+easyblock = 'CMakePythonPackage'
+
+name = 'DOLFINx-MPC'
+version = '0.9.0'
+
+homepage = 'https://github.com/jorgensd/dolfinx_mpc'
+description = "Implementation of multi-point constraints in DOLFINx."
+
+toolchain = {'name': 'foss', 'version': '2023b'}
+
+source_urls = ['https://github.com/jorgensd/dolfinx_mpc/archive/refs/tags/']
+sources = ['%(name)s-%(version)s.tar.gz']
```

it is https://github.com/jorgensd/dolfinx_mpc/archive/refs/tags/v0.9.0.tar.gz
```suggestion
sources = ['v%(version)s.tar.gz']
```